client/my-sites/plan-features/item.jsx

```diff
@@ -9,31 +9,48 @@ import classNames from 'classnames';
  */
 import Gridicon from 'components/gridicon';
 import InfoPopover from 'components/info-popover';
+import { abtest } from 'lib/abtest';
 
-export default function PlanFeaturesItem( { description, children, className } ) {
-	const renderTipinfo = () => {
-		if ( ! description ) {
-			return null;
-		}
+//export default function PlanFeaturesItem( { description, children, className } ) {
+export default React.createClass( {
+
+	displayName: 'PlanFeaturesItem',
 
+	getInitialState() {
+		return {
+			showPopover: false
+		};
+	},
+
+	renderTipinfo( description ) {
 		return (
 			<div className="plan-features__item-tip-info">
 				<InfoPopover
 					position="right"
+					open = { this.state.showPopover }
 				>
 					{ description }
 				</InfoPopover>
 			</div>
 		);
-	};
-
-	const classes = classNames( 'plan-features__item', className );
+	},
+	render() {
+		const { description, children, className } = this.props;
+		const classes = classNames( 'plan-features__item', className );
+		let popoverBindings = {};
+		if( abtest( 'plansDescriptions' ) === 'auto' ) {
+			popoverBindings = {
+				onMouseOver: () => this.setState( { showPopover: true } ),
+				onMouseOut: () => this.setState( { showPopover: false } )
+			};
```

Comment:
JSX props should not use arrow functions because it creates a new function each time you hover the mouse over/out. Better approach is to bind a class method to `this` of the class in `constructor`.